src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
+
+const widenAncestors = (el) => {
+  while ((el = el.parentElement) && !el.classList.contains('content_block')) {
+    // Only widen if there was a width setting. Keeps changes minimal.
+    if (el.style.width) {
+      el.style.width = '100%'
+    }
+    if (el.style.maxWidth) {
+      el.style.maxWidth = '100%'
+    }
+    if (el.style.float) {
+      el.style.float = 'none'
+    }
+  }
+}
+
+const shouldWidenImage = (image) => {
+  // Some images are within a <div class='noresize'>...</div> which indicates
+  // they should not be widened. Example below has links overlaying such an image.
+  // See:
+  //   'enwiki > Counties of England > Scope and structure > Local government'
+  if (utilities.findClosest(image, "[class*='noresize']")) {
```

Comment:
..I didn't know about `closest()`. It sounds much _closer_ to the functionality provided so +1 for keeping the word "closest" in the name. I don't really mind if you do or don't tack other words on.